app/presenters/admin_auction_status_presenter/available.rb

```diff
@@ -4,7 +4,26 @@ def header
   end
 
   def body
-    I18n.t('statuses.bid_status_presenter.available.admin.body', end_date: end_date)
+    if total_bids > 0
```

Comment:
Yeah, I assume you mean moving this into the `AdminAuctionStatusPresenterFactory` and `BidStatusPresenter` factories and have them look at the number of bids? I thought about that, but I kinda want to limit the factories to only needing to know about the status fields (I would probably merge the two `Future` presenters by this same logic.